--- node_modules/nodejs-mobile-react-native/android/src/main/cpp/native-lib.cpp.backup	2022-03-16 15:58:16.668392410 +0100
+++ node_modules/nodejs-mobile-react-native/android/src/main/cpp/native-lib.cpp	2022-03-16 15:58:42.764334863 +0100
@@ -148,6 +148,7 @@
     //Set the builtin_modules path to NODE_PATH.
     const char* path_path = env->GetStringUTFChars(modulesPath, 0);
     setenv("NODE_PATH", path_path, 1);
+    setenv("DEBUG", "backend:*", 1);
     env->ReleaseStringUTFChars(modulesPath, path_path);
 
     //argc
