--- node_modules/nodejs-mobile-react-native/ios/NodeRunner.mm.backup	2022-03-16 17:44:25.000000000 +0100
+++ node_modules/nodejs-mobile-react-native/ios/NodeRunner.mm	2022-03-16 17:33:24.000000000 +0100
@@ -205,6 +205,7 @@
     nodePath = [nodePath stringByAppendingString:builtinModulesPath];
   }
   setenv([@"NODE_PATH" UTF8String], (const char*)[nodePath UTF8String], 1);
+  setenv([@"DEBUG" UTF8String], "waggle:*", 1);
 
   int c_arguments_size=0;
 
