--- node_modules/nodejs-mobile-react-native/android/src/main/cpp/native-lib.cpp.backup	2022-08-31 11:49:47.331103861 +0200
+++ node_modules/nodejs-mobile-react-native/android/src/main/cpp/native-lib.cpp	2022-08-31 11:50:07.430981315 +0200
@@ -67,9 +67,9 @@
 void rcv_message(const char* channel_name, const char* msg) {
   JNIEnv *env=cacheEnvPointer;
   if(!env) return;
-  jclass cls2 = env->FindClass("com/janeasystems/rn_nodejs_mobile/RNNodeJsMobileModule");  // try to find the class
+  jclass cls2 = env->FindClass("com/zbaymobile/NotificationModule");  // try to find the class
   if(cls2 != nullptr) {
-    jmethodID m_sendMessage = env->GetStaticMethodID(cls2, "sendMessageToApplication", "(Ljava/lang/String;Ljava/lang/String;)V");  // find method
+    jmethodID m_sendMessage = env->GetStaticMethodID(cls2, "notify", "(Ljava/lang/String;Ljava/lang/String;)V");  // find method
     if(m_sendMessage != nullptr) {
         jstring java_channel_name=env->NewStringUTF(channel_name);
         jstring java_msg=env->NewStringUTF(msg);
