--- node_modules/factory-girl/package.json	2022-08-10 08:37:01.488216497 +0200
+++ node_modules/factory-girl/package.json.backup	2022-08-10 08:42:49.621878301 +0200
@@ -1,7 +1,7 @@
 {
   "name": "factory-girl",
   "description": "A factory library for Node.js and JavaScript inspired by factory_girl",
-  "main": "./src/index",
+  "main": "index.js",
   "author": [
     "@aexmachina",
     "@chetanism"
@@ -32,4 +32,4 @@
     "babel-runtime": "^6.11.6",
     "chance": "^1.0.4"
   }
-}
+}
\ No newline at end of file
