--- node_modules/classic-level/index.backup.js	2023-01-25 18:21:31.155350735 +0100
+++ node_modules/classic-level/index.js	2023-01-25 18:21:57.191327021 +0100
@@ -4,7 +4,7 @@
 const ModuleError = require('module-error')
 const { fromCallback } = require('catering')
 const fs = require('fs')
-const binding = require('./binding')
+const binding = require('./binding').default
 const { ChainedBatch } = require('./chained-batch')
 const { Iterator } = require('./iterator')
 
