--- node_modules/classic-level/iterator.backup.js	2023-01-25 18:22:41.707286685 +0100
+++ node_modules/classic-level/iterator.js	2023-01-25 18:23:28.191244822 +0100
@@ -1,7 +1,7 @@
 'use strict'
 
 const { AbstractIterator } = require('abstract-level')
-const binding = require('./binding')
+const binding = require('./binding').default
 
 const kContext = Symbol('context')
 const kCache = Symbol('cache')
