diff --git a/packages/backend/node_modules/electron-fetch/lib/index.js b/packages/backend/node_modules/electron-fetch/lib/index.js
index c9ea590d9..74c0757a8 100644
--- a/packages/backend/node_modules/electron-fetch/lib/index.js
+++ b/packages/backend/node_modules/electron-fetch/lib/index.js
@@ -1282,12 +1282,7 @@ function getNodeRequestOptions(request) {
 }
 
 let electron; // istanbul ignore else
-
-if (process.versions.electron) {
-  electron = require('electron');
-}
-
-const isReady = electron && electron.app && !electron.app.isReady() ? new Promise(resolve => electron.app.once('ready', resolve)) : Promise.resolve();
+const isReady = Promise.resolve();
 /**
  * Fetch function
  *
