--- node_modules/ipfs-unixfs-exporter/src/resolvers/unixfs-v1/content/file_backup.js	2023-02-28 15:23:36.182726959 +0100
+++ node_modules/ipfs-unixfs-exporter/src/resolvers/unixfs-v1/content/file.js	2023-02-28 15:19:24.258946254 +0100

@@ -150,7 +150,7 @@
     // use a queue to walk the DAG instead of recursion to ensure very deep DAGs
     // don't overflow the stack
     const walkQueue = new PQueue({
-      concurrency: 1
+      concurrency: 100000
     })
     const queue = pushable()
 
