--- node_modules/pvutils/build/utils.js.backup
+++ node_modules/pvutils/build/utils.js
@@ -13,7 +13,7 @@
     if ((parameters instanceof Object) === false) {
         return defaultValue;
     }
-    return parameters[name] ?? defaultValue;
+    return parameters[name] || defaultValue;
 }
 function bufferToHexCodes(inputBuffer, inputOffset = 0, inputLength = (inputBuffer.byteLength - inputOffset), insertSpace = false) {
     let result = "";
--- node_modules/parse-duration/index.js.backup
+++ node_modules/parse-duration/index.js
@@ -1,6 +1,6 @@
 'use strict'
 
-var durationRE = /(-?(?:\d+\.?\d*|\d*\.?\d+)(?:e[-+]?\d+)?)\s*([\p{L}]*)/uig
+var durationRE = /(-?(?:\d+\.?\d*|\d*\.?\d+)(?:e[-+]?\d+)?)\s*([\p{L}]*)/ig
 
 module.exports = parse
 // enable default import syntax in typescript
