--- node_modules/orbit-db-keystore/node_modules/node-gyp-build/index.js.backup	2022-08-25 15:09:08.382867038 +0200
+++ node_modules/orbit-db-keystore/node_modules/node-gyp-build/index.js	2022-08-25 15:09:25.942845956 +0200
@@ -29,7 +29,10 @@
   } catch (err) {}
 
   if (!prebuildsOnly) {
-    var release = getFirst(path.join(dir, 'build/Release'), matchBuild)
+    var split = dir.split('/')
+    var package = split[split.length - 1]
+
+    var release = getFirst(path.join(dir, `../../../${package}/build/Release`), matchBuild)
     if (release) return release
 
     var debug = getFirst(path.join(dir, 'build/Debug'), matchBuild)
