--- node_modules/electron-fetch/lib/index.js	2023-02-02 10:04:57.162302045 +0100
+++ node_modules/electron-fetch/lib/index.backup.js	2023-02-02 10:16:12.140781319 +0100
@@ -1283,11 +1283,7 @@
 
 let electron; // istanbul ignore else
 
-if (process.versions.electron) {
-  electron = require('electron');
-}
-
-const isReady = electron && electron.app && !electron.app.isReady() ? new Promise(resolve => electron.app.once('ready', resolve)) : Promise.resolve();
+const isReady = Promise.resolve();
 /**
  * Fetch function
  *
