--- node_modules/ipfs-pubsub-peer-monitor/src/ipfs-pubsub-peer-monitor.js	2024-05-08 12:44:48
+++ node_modules/ipfs-pubsub-peer-monitor/src/ipfs-pubsub-peer-monitor.backup.js	2024-05-08 12:44:25
@@ -55,7 +55,7 @@
   async _pollPeers () {
     try {
       const peers = await this._pubsub.peers(this._topic)
-      IpfsPubsubPeerMonitor._emitJoinsAndLeaves(new Set(this._peers), new Set(peers), this)
+      IpfsPubsubPeerMonitor._emitJoinsAndLeaves(new Set(this._peers.map(p => p.toString())), new Set(peers.map(p => p.toString())), this)
       this._peers = peers
     } catch (err) {
       clearInterval(this._interval)