--- node_modules/parse-duration/index.js	2022-02-24 18:19:51.000000000 +0100
+++ node_modules/parse-duration/index.js.backup	2022-02-24 18:19:26.000000000 +0100
@@ -1,6 +1,6 @@
 'use strict'
 
-var durationRE = /(-?(?:\d+\.?\d*|\d*\.?\d+)(?:e[-+]?\d+)?)\s*([\p{L}]*)/uig
+var durationRE = /(-?(?:\d+\.?\d*|\d*\.?\d+)(?:e[-+]?\d+)?)\s*([\p{L}]*)/ig
 
 module.exports = parse
 // enable default import syntax in typescript
